### DISCUSSION TITLE
Questions regarding neural_prophet usages.

### DISCUSSION THREAD

**Discussion by bergen288** [ourownstory/neural_prophet#350] [Q&A - get help using NeuralProphet] [answered]
neural_prophet only gives me the plot with forecast value.  Is it possible to make make plot with history and forecast values?
The following add_seasonality works fine in fbprophet, but it fails with neural_prophet due to "unexpected keyword argument 'condition_name'".  What's the correct way to define it?
```
model.add_seasonality(name='daily_weekday',
                      period=1,
                      fourier_order=3,
                      condition_name='weekday')
```

**Comment by ourownstory** [comment, marked as answer]
Hi @bergen288 

> neural_prophet only gives me the plot with forecast value. Is it possible to make make plot with history and forecast values?

Yes, you can use ```plot_last_forecast(plot_history_data=True)```

> The following add_seasonality works fine in fbprophet, but it fails with neural_prophet due to "unexpected keyword argument 'condition_name'". What's the correct way to define it?

I am sorry to say that conditional seasonality is not implemented yet.

**Reply by olafsrb** [reply]
Is m.add_seasonality still not implemented in NP? 
regards,

**Reply by ourownstory** [reply]
hi @olafsrb 
`m.add_seasonality` is implemented. 
`condition_name` subfunctionality is currently not implemented.
